src/components/pages/InstructorHome/InstructorSidebar.js

```diff
@@ -1,43 +1,52 @@
 import { Layout, Menu } from 'antd';
-import React from 'react';
+import React, { useState } from 'react';
 import {
   DesktopOutlined,
-  PieChartOutlined,
-  FileOutlined,
+  ToolOutlined,
   TeamOutlined,
   UserOutlined,
   MailOutlined,
+  HomeOutlined,
+  ExportOutlined,
+  ThunderboltOutlined,
 } from '@ant-design/icons';
 
 const { SubMenu } = Menu;
 const { Sider } = Layout;
 
 function InstructorSidebar() {
+  const [collapsed, setCollapsed] = useState(false);
+
+  const onCollapse = () => {
+    if (collapsed === true) {
+      setCollapsed(false);
+    } else {
+      setCollapsed(true);
+    }
+  };
+
   return (
     <Layout style={{ minHeight: '100vh' }}>
-      <Sider>
+      <Sider collapsible collapsed={collapsed} onCollapse={onCollapse}>
```

Comment:
This collapsing looks AMAZING in your loom video - super blown away, nice job! @andrewskr90 @Jie-chelchel